aiodav/__init__.py,sha256=yS8J1sjRTkvDEynXPGmAoxc0GpIS3wiljNgqiWCZoTc,101
aiodav/client.py,sha256=_YnLhcC9kAa3tFLlouJ38iODUFXPbWBvDT3ov7ByiNY,47474
aiodav/exceptions.py,sha256=FS7Dfal-rMLA4T-9Vavx8_IxqAiWVYkATND-GSysWB0,2559
aiodav/urn.py,sha256=2Hf0tyHPzKEy227qS9ET8eKXI8sf-fHI6K2ISRWT1CU,2100
aiodav/utils.py,sha256=zXEs6nPNtxvaGdYeUauRJZBZm5SmAyQ0qR3i7Ft2FIA,10689
aiodav-0.1.10.dist-info/LICENSE,sha256=m6FQVED-O2rDQzzTx89dp7PYYlfEd-aUGzfO7xqS9To,1084
aiodav-0.1.10.dist-info/METADATA,sha256=gAP9ARJmo9e1NYfxy4kmb24f-uSuopCOROmWkjRDH6M,2474
aiodav-0.1.10.dist-info/WHEEL,sha256=G16H4A3IeoQmnOrYV4ueZGKSjhipXx8zc8nu9FGlvMA,92
aiodav-0.1.10.dist-info/top_level.txt,sha256=izb6vrcNoTNHhg5DB4BLgTD7d_m6lXBZo9dmeD4zjDQ,7
aiodav-0.1.10.dist-info/RECORD,,
